chain-config/src/lib.rs

```diff
@@ -78,13 +81,66 @@ pub trait ChainConfigContract:
         self.execute_bridge_operation_event(&hash_of_hashes, &config_hash);
     }
 
+    #[endpoint(resumeRegistration)]
+    fn update_registration_status(&self, hash_of_hashes: ManagedBuffer, registration_status: u8) {
+        self.require_setup_complete();
+
+        let status_hash = ManagedBuffer::new_from_bytes(
+            &self
+                .crypto()
+                .sha256(ManagedBuffer::new_from_bytes(&[registration_status]))
+                .to_byte_array(),
+        );
+
+        if registration_status != 0 && registration_status != 1 {
+            self.failed_bridge_operation_event(
+                &hash_of_hashes,
+                &status_hash,
+                &ManagedBuffer::from(INVALID_REGISTRATION_STATUS),
+            );
+
+            self.remove_executed_hash(&hash_of_hashes, &status_hash);
+
+            return;
+        }
+
+        self.lock_operation_hash(&status_hash, &hash_of_hashes);
+
+        let registration_status_mapper = self.registration_status();
+
+        if registration_status == registration_status_mapper.get() {
```

Comment:
I don't think this check is really necessary, it's just more code and more tests not needed